registry_quickstart/administrators/system_configuration.adoc

```diff
@@ -128,6 +131,16 @@ Restart the origin service to update the running configuration.
 $ sudo docker restart origin
 ----
 
```

Comment:
Please add an anchor right before the heading to ensure that the link above works properly:

[[using-service-account-tokens-for-authentication]]
== Using Service Account Tokens for Authentication